saleor/graphql/product/mutations/products.py

```diff
@@ -1890,4 +1903,5 @@ def perform_mutation(cls, _root, info, **data):
             product.available_for_purchase = start_date
 
         product.save(update_fields=["available_for_purchase", "updated_at"])
+        info.context.plugins.product_updated(product)
```

Comment:
We should prefetch `variants` in this mutation.

Reply:
yep, im adding that as we speak